ProcessMaker/Cache/Settings/SettingCacheManager.php

```diff
@@ -168,14 +168,11 @@ public function clear(): bool
      */
     public function clearBy(string $pattern): void
     {
-        $defaultDriver = $this->manager->getDefaultDriver();
-
-        if ($defaultDriver !== 'cache_settings') {
-            throw new SettingCacheException('The cache driver must be Redis.');
-        }
-
+        //get the default driver
+        $defaultDriver = config('cache.stores.cache_settings.connection') ?? 'redis';
         try {
-            $prefix = $this->manager->getPrefix();
+            //get the prefix
+            $prefix = config('cache.stores.cache_settings.prefix', 'settings:');
```

Comment:
Use a constant for the default prefix
e.g. const DEFAULT_PREFIX = 'settings:';

```suggestion
            $prefix = config('cache.stores.cache_settings.prefix', self::DEFAULT_PREFIX);
```